### PR TITLE
[ticket/15569] Suggest using file replacing instead of automatic updater

### PR DESCRIPTION
**Pull request by marc1706**
PHPBB3-15569

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15569

### CODE REVIEW

File: phpBB/language/en/install.php [hunk 1]
```diff
@@ -355,17 +355,40 @@
...
 
+		<h1>How to update your installation with the Full Package</h1>
+
+		<p>The recommended way of updating your installation is using the full package. If core phpBB files have been modified in your installation you may wish to use the automatic update package in order to not lose these changes. You are also able to update your installation using the other methods listed within the INSTALL.html document. The steps for updating phpBB3 using the full package are:</p>
+
+		<ol style="margin-left: 20px; font-size: 1.1em;">
+			<li><strong class="error">Backup all board files and the database.</strong></li>
+			<li>Go to the <a href="https://www.phpbb.com/downloads/" title="https://www.phpbb.com/downloads/">phpBB.com downloads page</a> and download the latest "Full Package" archive.</li>
+			<li>Unpack the archive.</li>
+			<li>Remove (delete) the <code class="inline">config.php</code> file, and the <code class="inline">/images</code>, <code class="inline">/store</code> and <code class="inline">/files</code> folders <em>from the package</em> (not your site).</li>
```
Comment (Crizz0) [inline phpBB/language/en/install.php]: why do we recommend to remove the images/, too? Adding new images e. g. smilies won't be recognized by updaters.

Comment (marc1706) [reply phpBB/language/en/install.php]: It's the same text as here so I'd actually keep it like that: https://www.phpbb.com/support/docs/en/3.2/ug/upgradeguide/upgrade31/

We might employ emojis more often so I'd say it's ok like that.

Comment (Crizz0) [reply phpBB/language/en/install.php]: I'm. fine with it.

**Comment by Solidjeuh** [comment]
And what about the store folder? It contains backups made via FTP
https://github.com/phpbb/phpbb/pull/5147/files#diff-f302393b943b77ad2acfb8ee3dbe3b6cR274

**Pull request merged**